constants.xml

```diff
@@ -318,7 +318,7 @@
         <ENABLE_CHECK_PERFORMANCE_LOG>false</ENABLE_CHECK_PERFORMANCE_LOG>
         <NUM_TXN_TO_SEND_PER_ACCOUNT>100</NUM_TXN_TO_SEND_PER_ACCOUNT>
         <ENABLE_ACCOUNTS_POPULATING>false</ENABLE_ACCOUNTS_POPULATING>
-        <UPDATE_PREGENED_ACCOUNTS>false</UPDATE_PREGENED_ACCOUNTS>
+        <UPDATE_PREGENED_ACCOUNTS>true</UPDATE_PREGENED_ACCOUNTS>
```

Comment:
I do not know why this value has been changed, where and why did this change happen?